Roys/app/src/main/java/com/dwg_karrier/roys/ScriptedURL.java

```diff
@@ -42,7 +47,16 @@ public boolean getIsRead() {
     return isRead;
   }
 
+  public String getTitle() {
+    return crawler.getTitle();
+  }
+
+  public String getContent() {
+    return crawler.getContent();
+  }
+
   public int getWordCount() {
+    wordCount = crawler.getWordCount();
```

Comment:
Storing the value is a good idea, but current getter design makes storing meaningless.
Getter pattern is usually the only exposed way for outsiders to read the variable, and this getter calls `crawl.getWordCount()` every time, which makes no way to use `wordCount` directly.
One of the suggestion my suggestion is to add `word = crawler.getWordCount();` at the end of the constructor and just do `return wordCount;` in `getWordCount` so you can benefit of storing the value to reduce the time.